### ISSUE TITLE
Delete below 2 users info from firebase and database.

### ISSUE THREAD

**Issue by GochaLearningSpace** [Willings-Inc/SCG#178]
**User 1 :** 
- Name:　ああああ　ああああ
- email address　[parasekuto888@gmail.com](mailto:parasekuto888@gmail.com)
- password　gjgj4545
- telephone number　08037594794 （編集済み）

**User 2** 
- [saiwaki@daraf.co.jp](mailto:saiwaki@daraf.co.jp)

**Comment by amanscisingh** [comment]
@GochaLearningSpace I investigated. there is no data for userparasekuto888@gmail.com and saiwaki@daraf.co.jp in the database.
And in Firebase there is data of only parasekuto888@gmail.com, no data for saiwaki@daraf.co.jp.
So should I proceed with deleting the data of parasekuto888@gmail.com from FireBase?

**Comment by GochaLearningSpace** [comment]
> @GochaLearningSpace I investigated. there is no data for [userparasekuto888@gmail.com](mailto:userparasekuto888@gmail.com) and [saiwaki@daraf.co.jp](mailto:saiwaki@daraf.co.jp) in the database. And in Firebase there is data of only [parasekuto888@gmail.com](mailto:parasekuto888@gmail.com), no data for [saiwaki@daraf.co.jp](mailto:saiwaki@daraf.co.jp). ![image](https://user-images.githubusercontent.com/68449680/239103409-330a34f9-0eaa-48f2-a740-e2e403e4cd94.png)
> 
> So should I proceed with deleting the data of [parasekuto888@gmail.com](mailto:parasekuto888@gmail.com) from FireBase?

yes, please delete, by the way, I thought I wrote [parasekuto888@gmail.com](mailto:parasekuto888@gmail.com) from beginning. Can I ask why you thought you needed delete, [userparasekuto888@gmail.com](mailto:userparasekuto888@gmail.com) ?

**Comment by amanscisingh** [comment]
@GochaLearningSpace It was a typing mistake that I wrote userparasekuto888@gmail.com.
I have deleted the user.

**Comment by GochaLearningSpace** [comment]
@amanscisingh 

Ok thank you for letting me know.
